test/dlc_tests/ecs/tensorflow/training/test_ecs_tensorflow_training.py

```diff
@@ -29,16 +30,16 @@ def test_ecs_tensorflow_training_mnist_cpu(cpu_only, ecs_container_instance, ten
     ecs_utils.ecs_training_test_executor(ecs_cluster_name, cluster_arn, training_cmd, tensorflow_training, instance_id)
 
 
-@pytest.mark.model("mnist")
-@pytest.mark.parametrize("training_script", [TF_MNIST_TRAINING_SCRIPT], indirect=True)
+@pytest.mark.model("FasterRCNN")
+@pytest.mark.parametrize("training_script", [TF_FasterRCNN_TRAINING_SCRIPT], indirect=True)
 @pytest.mark.parametrize("ecs_instance_type", ["p2.8xlarge"], indirect=True)
```

Comment:
Does this test need a p2.8xlarge with 8 GPUs? Can it use a smaller instance type or a smaller number of GPUs?

Reply:
Yeah, but default the instance type is p2.8xlarge, and I didn't change that, I think we should keep the same with the other tests